### DISCUSSION TITLE
windows alist使用frp内网穿透无法在外网访问。

### DISCUSSION THREAD

**Discussion by ukzshd** [AlistGo/alist#4189] [Q&A]
已经在本地搭建了alist，可以运行
<img width="604" alt="截屏2023-04-18 19 49 13" src="https://user-images.githubusercontent.com/96689369/232768142-4ecd250a-d9c9-4361-be08-2980d8762308.png">
局域网内也能访问。
然后使用frp进行内网穿透
<img width="977" alt="截屏2023-04-18 19 51 34" src="https://user-images.githubusercontent.com/96689369/232768634-1bb0eff8-feb1-48d6-890a-21aa3a550e62.png">
穿透了两个服务的端口，qb和alist。
在外网中，qb可以正常访问，说明内网穿透是正常的
![截屏2023-04-18 19 53 38](https://user-images.githubusercontent.com/96689369/232769396-ab100b43-ec15-4531-9a9e-8d9927c78763.png)
但是alist在外网无法打开
![截屏2023-04-18 19 56 10](https://user-images.githubusercontent.com/96689369/232770048-75921df6-b448-4574-a821-a586a9f74986.png)
我记得以前在openwrt安装里面有允许外网访问，window下没有找到相关设置，不知道如何解决，求助，感谢大佬帮忙解决

**Comment by ifdelete** [comment]
应该还是出在frp上，用内穿的时候说不需要考虑防火墙的，毕竟直接是一个二级域名。

**Comment by hhmmLife** [comment]
我也遇到了这个问题 我使用的是zeroTier

**Reply by hhmmLife** [reply]
在我windows防火墙关闭之后可以穿透访问了，但是在这之前我的hfs使用的80端口是可以正常访问的，不知道为什么
